.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish package to PyPI
+
+on:
+  push:
+    tags: 
+      - '*'
+
+jobs:
+  push:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install pip
+        run: pip install -r requirements/pip.txt
+
+      - name: Build package
+        run: python setup.py sdist bdist_wheel
+
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@master
```

Comment:
Just passing by :)

Nit: the `master` branch of this action [has been sunset](https://github.com/pypa/gh-action-pypi-publish#-master-branch-sunset). We should use `release/v1` instead. We could also update the `checkout` to `v3` and `setup-python` to `v4`.

Side note: I personally like adding a Python package check to the CI to verify if the `pypi-publish` action will succeed (it can fail even due to bad formatting of the README or the changelog). You can do this by adding the following snippet to `tox.ini`:
```ini
[testenv:package]
deps =
    build
    twine
commands =
    python -m build
    twine check dist/*
```

Then, adding it to the CI is only a matter of adding `package` to the `toxenv` in `.github/workflows/ci.yml`.

Reply:
Brilliant @Agrendalath , thank you for your timely suggestions :) Addressed with [acbb66f](https://github.com/openedx/openedx-learning/pull/65/commits/acbb66f0ec9e65121a47fcde4bde1e1b8185a3c5).